### PR TITLE
Sidebar: Align contents properly

### PR DESCRIPTION
**Pull request by dennisreimann**
Some pixel nudging as well as adjusting the mobile sidebar, so that the toggle does not need an extra HTML element.

sidebar

### CODE REVIEW

**Comment by ben-kaufman** [comment]
Hi, thanks for the PR! I'm seeing though that now the sidebar with the > symbol is appearing even on mobile, covering up the menu button
Screen Shot 2020-09-16 at 9 01 27

**Comment by dennisreimann** [comment]
@ben-kaufman Thanks for the pointer, my bad, missed that case!

File: src/cryptoadvance/specter/static/styles.css [hunk 1]
```diff
@@ -908,38 +909,46 @@ a:focus + .tool-tip .tool-tip__info {
...
     }
     #side-content:hover{
-      position: absolute;
-      left: 0px;
+      left: 0;
     }
-    #side-content:hover > #side-expand{
-      transform: scaleX(-1);
+    #side-content:hover:after{
+      right: -20px;
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/static/styles.css:944]:
```suggestion
      position: absolute;
      right: -20px;
```

File: src/cryptoadvance/specter/static/styles.css [hunk 2]
```diff
@@ -908,38 +909,46 @@ a:focus + .tool-tip .tool-tip__info {
...
-  #side-expand{
+  #side-content:after{
     display: flex;
+    content: "▶";
     align-items: center;
     justify-content: center;
-    font-size: 14px;
+    font-size: 12px;
     cursor: pointer;
+    position: absolute;
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/static/styles.css:926]:
```suggestion
    position: sticky;
```

**Review by ben-kaufman — COMMENTED** [review]
Looks good, just a small fix to ensure the arrow stays in place even in case the list of wallet and devices is long and the user scrolls down on the sidebar.

File: src/cryptoadvance/specter/static/styles.css (hunk 1 above)
Comment (dennisreimann) [reply src/cryptoadvance/specter/static/styles.css:944]: It already is positioned absolutely via [line 926](https://github.com/cryptoadvance/specter-desktop/pull/394/files/ae2fc0ddd2007f952f5292ad61847a8c4bf451fd#diff-51788d3f42cf0703ce17eb762b3b4f1bR926).

Comment (ben-kaufman) [reply src/cryptoadvance/specter/static/styles.css:944]: Yes but now that it's sticky the arrow remains visible even on hover (unless that's intended?)

Comment (dennisreimann) [reply src/cryptoadvance/specter/static/styles.css:945]: You're right, thanks again.

**Review by ben-kaufman — APPROVED** [review]

**Comment by ben-kaufman** [comment]
Looks good now, thanks! Merging.

**Pull request merged**